Library/Homebrew/utils/analytics.sh

```diff
@@ -113,6 +113,8 @@ report-analytics-screenview-command() {
       "${args[@]}" \
       --silent --output /dev/null &>/dev/null & disown
   else
+    echo "$HOMEBREW_CURL https://www.google-analytics.com/debug/collect" \
+      "${args[@]}"
     "$HOMEBREW_CURL" https://www.google-analytics.com/debug/collect \
       "${args[@]}"
```

Comment:
I like how you handled this in the Ruby code. How about doing something similar here?

```shell
local url="https://www.google-analytics.com/debug/collect"
echo "$HOMEBREW_CURL $url ${args[*]}"
"$HOMEBREW_CURL" "$url" "${args[@]}"
```

(Note how I used both `[*]` and `[@]` depending on context. The former, if it appears inside a string, is essentially equivalent to `args.join(" ")` in Ruby. The latter expands to individual arguments, a bit like `*args` in Ruby.)